### DISCUSSION TITLE
Anyone using riverpod in production/open-source apps?

### DISCUSSION THREAD

**Discussion by msarkrish** [rrousselGit/riverpod#1350] [Show and tell]
Similar to:
https://github.com/rrousselGit/provider/issues/62

If anyone is using the library then please give your input.

Use the following template:

If available on stores
iOS:
Android:

If open source
Source Code:

Your opinion
Version: ?
Years of experience: ?
Good: ?
Bad: ?

Thank you very much!

**Comment by rrousselGit** [comment]
Let's convert this to a discussion instead, now that that's a thing.

**Reply by msarkrish** [reply]
Can you pin the issue and discussions? @rrousselGit so that others can see this and they can add answer.

**Comment by FXschwartz** [comment]
I have two private business apps in production for Android & iOS, and one Web & Desktop app I'm currently working on that will go into production in the next 3 months all using Riverpod for state management. I have another app in Production that uses Provider and I am planning on migrating that app over to Riverpod in the next 6 months.

7 years of developer experience, 2 years of Flutter experience.

There are a lot of good things about this package, the flexibility to orchestrate your providers and app state the way you want, the ease of creating a new provider, and the speed of development. The community is extremely helpful and active. There are more and more examples being added to the website to show how to implement different use cases.

The main negative I can think of is just the flipside to using something so flexible. Because there isn't a "one way" to do things with Riverpod if you are just learning it, it can be a bit confusing what the best way is to set up your providers. The best way to mitigate this is just to start using them in a POC app or project so you can get a handle on how they all work. The documentation is getting better and better and more examples are getting added.

**Comment by titomitto** [comment]
Hey devs! I recently created a Todo app that uses Clean Architecture and Riverpod. You can check out the code here: https://github.com/TitoMitto/clean-todo. I would love to get your feedback and suggestions on how to improve it. The app is built with a focus on separation of concerns, testability, and maintainability, making it easy to understand and modify. So, feel free to take a look and let me know what you think!

**Comment by noriHanda** [comment]
Finally after months I've released this video editor app for my client.
It is meant to easily make an one-minute-intro video of your academic research.

It's not fully i18ned yet, and it still has some bugs related to Android MediaX, but it should work in most cases.
It's a case of riverpod in use in education field.

iOS: https://apps.apple.com/app/id6466738852
Android: https://play.google.com/store/apps/details?id=ac.jp.hokudai.Rakulip

Version: 3.0.0-dev.0
Years of experience: 2.5
Good: Heck yeah!
Bad: What is?

**Comment by arafaysaleem** [comment]
The following apps use riverpod but they are made with earlier version and I haven't updated them. The latest ones are private.

www.github.com/arafaysaleem/ez_tickets_app
www.github.com/arafaysaleem/stadium_tickets_app
www.github.com/cyntaria/UniPal-Frontend

**Comment by SalehTZ** [comment]
Hiddify is using Riverpod:
https://github.com/hiddify/hiddify-app
